templates/masters-conference/index.html

```diff
@@ -11,35 +11,79 @@
   <div class="row u-equal-height u-vertically-center">
     <div class="col-6">
       <h1>Ubuntu Masters:<br />the masters speak</h1>
-      <p>Ubuntu Masters are IT practitioners who are driving revolutionary change in the corporate space. Watch videos from our first Ubuntu Masters Conference, featuring these inspiring doers to hear how they are solving complex industry-wide challenges.</p>
+      <p>Ubuntu Masters are IT practitioners who are driving revolutionary change in the corporate space. Watch videos from previous Ubuntu Masters speakers, featuring inspiring doers who are solving complex industry-wide challenges. And, sign up for future events and content updates.</p>
 
       <p>
-        <a href="/masters-conference/contact-us" class="p-button--positive">Get in touch</a>
+        <a href="/masters-conference/contact-us" class="p-button--positive">
+          Get in touch
+        </a>
       </p>
     </div>
     <div class="col-6 u-hide--small u-align--right">
       {{
         image(
-          url="https://assets.ubuntu.com/v1/4b34a715-ubuntu-masters-logo.svg",
-          alt="",
-          height="130",
-          width="300",
-          hi_def=True,
-          loading="auto",
-          attrs={"class": "u-hide--small"},
+        url="https://assets.ubuntu.com/v1/4b34a715-ubuntu-masters-logo.svg",
+        alt="",
+        height="130",
+        width="300",
+        hi_def=True,
+        loading="auto",
+        attrs={"class": "u-hide--small"},
         ) | safe
       }}
     </div>
   </div>
 </section>
 
+<section class="p-strip--light is-bordered">
+  <div class="row">
+    <div class="col-4">
+      <h2>
+        Ubuntu Masters Virtual Conference: 30 April 2020
+      </h2>
+    </div>
+    <div class="col-8">
+      <table>
+        <tr>
+          <th width="25%">11:00 am ET</th>
+          <td><strong>Opening Session</strong></td>
+        </tr>
+        <tr>
+          <th>11:30 pm ET</th>
+          <td>
+            <strong>Scaling years of growth in one week: Building a high-performance global internet edge using Ubuntu and open source</strong><br />
+            <cite><a class="p-link--external" href="https://www.linkedin.com/in/robwc/">Rob Cameron</a>, Technical Director, Cloud Services, Roblox</cite>
```

Comment:
The copy doc has "Technical Director, Cloud Services & Adam Mills, Principal Engineer, Roblox"